imports/react-ui/insights/containers/PunchCard.js

```diff
@@ -0,0 +1,26 @@
+import { compose } from 'react-komposer';
+import { Brands } from '/imports/api/brands/brands';
+import { getTrackerLoader, composerOptions } from '/imports/react-ui/utils';
+
+import { PunchCard } from '../components';
+import { Mongo } from 'meteor/mongo';
+
+const MainData = new Mongo.Collection('punch_card');
```

Comment:
Same here.

Why is this collection created here? Collections must be declared in `api`. See the other modules such as `customers` and `conversations`.